docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -50,6 +50,15 @@ For information about how to support multiple tenants, see xref:security-openid-
 
 == Using the authorization code flow mechanism
 
+=== Configuring the Quarkus application to redirect the user
+
+As outlined in the previous section, the Authorization Code Flow begins with the user being redirected to the OIDC provider for authentication.
+To enable this redirect, the `quarkus.oidc.application-type` property must be set to `web-app`. This tells Quarkus that the
+application serves pages (usually a front-end application).
```

Comment:
Quarkus security does not really know what the application does, may be we can indeed add some clarification, I'll try to comment more tomorrow
```suggestion
```

Reply:
How about, `Usually, the Quarkus OIDC `web-app` application type must be set when your Quarkus application is a frontend application which serves HTML pages.` ? Or something like that